tests/fixtures/organizations.yml

```diff
@@ -0,0 +1,494 @@
+http_interactions:
+- request:
+    method: get
+    uri: https://api.github.com/organizations?since=1
+    body:
+      encoding: ''
+      string: ''
+    headers:
+      Accept: application/json, text/xml, application/xml, */*
+      Content-Type: ''
+      Authorization: token af0e9f121c2b68b6957a9b325de64fe572b1fd9a
+  response:
+    status:
+      status_code: 502
```

Comment:
so in this cassette there is first a 502 and then a 200. in the test I use expect_message to make sure I get the 502 first.